config.yaml

```diff
@@ -10,36 +10,46 @@ enableInlineShortcodes: true
 languages:
   en:
     languageName: English
-    title: Falco — Container-native Runtime Security
+    title: Falco — Cloud-Native Runtime Security
     contentDir: content/en
     params:
       heroText:
         # Main hero title
-        title: Container Native Runtime Security
+        title: Cloud-Native Runtime Security
         # Hero subtitle (optional)
-        subtitle: Falco is an open source project for intrusion and abnormality detection for Cloud Native platforms such as Kubernetes, Mesosphere, and Cloud Foundry. Detect abnormal application behavior. Alert via Slack, Fluentd, NATS, and more. Protect your platform by taking action through serverless (FaaS) frameworks, or other automation. 
+        subtitle: Falco, the open source cloud-native runtime security project, is the defacto Kubernetes threat detection engine. Falco detects unexpected application behavior and alerts on threats at runtime.
         # Button text
         buttontext: Get started
-      sectionAbout:
-        title: About Falco.
-        summary: With more responsibility shifting left to developers and the opaque nature of containers, organizations require deeper insight into container activities. The Falco project was hatched to understand container behavior and protect your platform from possible malicious activity. Leveraging Sysdig’s open source Linux kernel instrumentation, Falco gains deep insight into system behavior. The rules engine can then detect abnormal activity in applications, containers, the underlying host, and the container platform. 
-      section1:
-        title: Key Features.
+      runtime:
+        title: Why Runtime Security?
+        intro: >-
+               Securing Kubernetes requires putting controls in place to detect unexpected behavior that could be malicious. Examples include:
+        items:
+          - example: Exploits of unpatched and new vulnerabilities
+          - example: Insecure configurations
+          - example: Leaked or weak credentials
+          - example: Insider threats
+        closing: Even when processes are in place for vulnerability scanning and implementing pod security and network policies, not every risk will be addressed. You still need mechanisms to confirm these security barriers are effective, as well as help to configure them and provide a last line of defense when they fail.
```

Comment:
You still need mechanisms to confirm these security barriers are effective,  help configure them, and provide with a last line of defense when they fail.